tmt/steps/execute/internal.py

```diff
@@ -243,12 +242,14 @@ def _test_output_logger(
     def execute(
             self,
             *,
-            test: Test,
-            guest: Guest,
+            invocation: TestInvocation,
             extra_environment: Optional[EnvironmentType] = None,
             logger: tmt.log.Logger) -> list[CheckResult]:
         """ Run test on the guest """
-        logger.debug(f"Execute '{test.name}' as a '{test.framework}' test.")
+
+        test, guest = invocation.test, invocation.guest
+
+        logger.debug(f"Execute '{invocation.test.name}' as a '{invocation.test.framework}' test.")
```

Comment:
`test` and `guest` are assigned above anyway so no need for this?

Reply:
+1, could  be

```suggestion
        logger.debug(f"Execute '{test.name}' as a '{test.framework}' test.")
```

for brewity